examples/crm/src/tasks/Task.tsx

```diff
@@ -60,131 +66,141 @@ export const Task = ({
-                            onClick={() => {
-                                update('tasks', {
-                                    id: task.id,
-                                    data: {
-                                        due_date: new Date(
-                                            Date.now() + 7 * 24 * 60 * 60 * 1000
-                                        )
-                                            .toISOString()
-                                            .slice(0, 10),
-                                    },
-                                    previousData: task,
-                                });
-                                handleClose();
+                            <MoreVertIcon />
+                        </IconButton>
+                        <Menu
+                            id="basic-menu"
+                            anchorEl={anchorEl}
+                            open={open}
+                            onClose={Menu}
+                            MenuListProps={{
+                                'aria-labelledby': 'basic-button',
                             }}
                         >
-                            Postpone to next week
-                        </MenuItem>
-                        <MenuItem onClick={handleEdit}>Edit</MenuItem>
-                        <MenuItem onClick={handleDelete}>Delete</MenuItem>
-                    </Menu>
-                </>
-            }
-            disableGutters
-            sx={{ pr: 3 }}
-        >
-            <ListItemButton
-                role={undefined}
-                onClick={handleCheck()}
-                dense
-                disabled={isUpdatePending}
-                sx={{
-                    pl: 0,
-                    pr: '0!important',
-                    pt: 0,
-                    pb: 0,
-                    alignItems: 'flex-start',
-                }}
+                            <MenuItem
+                                onClick={() => {
+                                    update('tasks', {
+                                        id: task.id,
+                                        data: {
+                                            due_date: new Date(
+                                                Date.now() + 24 * 60 * 60 * 1000
+                                            )
+                                                .toISOString()
+                                                .slice(0, 10),
+                                        },
+                                        previousData: task,
+                                    });
+                                    handleCloseMenu();
+                                }}
+                            >
+                                Postpone to tomorrow
+                            </MenuItem>
+                            <MenuItem
+                                onClick={() => {
+                                    update('tasks', {
```

Comment:
**suggestion:** since this hanlder is almost the same as the previous one, it would be interesting to have a shared handler for both